<?xml version="1.0"?>
<launch>
<group ns="popi">
 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="joy_node" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
    <param name="autorepeat_rate" value="100" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="right_trigger" value="5" type="int"/>
  <param name="left_trigger" value="2" type="int"/>
  <param name="button_A" value="0" type="int"/>
  <param name="button_B" value="1" type="int"/>
  <param name="button_Y" value="3" type="int"/>
  <param name="button_LB" value="4" type="int"/>
  <param name="button_RB" value="5" type="int"/>

  <param name="scale_linear" value="0.01" type="double"/>
  <param name="simu" value="false" type="bool"/>
  
  <node pkg="popi_robot" type="manette_yannis" name="manette_yannis"/>
  </group>
</launch>